### ISSUE TITLE
Loss is inf

### ISSUE THREAD

**Issue by allipilli-harshitha** [buriburisuri/speech-to-text-wavenet#124]
hi @buriburisuri I got the same error for the first epoch i.e., loss = inf. what could be the reason for this infinity loss. could you please suggest what need to be changed.
initially my training has stopped in the first epoch after 92%. so i changed few parameters in sg_ctc. preprocess_collapse_repeated=True and ignore_longer_outputs_than_inputs=True.
After changing these parameters, training for epoch 1 is 100% but loss is inf.
